rackspace-server-manager.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+'''
+A script to create and delete the number of servers on Rackspace
+'''
+
+import pyrax
+import time
+import re
+import os
+import argparse
+from Crypto.PublicKey import RSA
+
+def create_key():
```

Comment:
It's probably best if this script didn't create the ssh key. Let's have this script take a path to SSH public to use for generating the machines.